gui/default/index.html

```diff
@@ -908,6 +908,10 @@ <h4 class="panel-title">
                           </span>
                         </td>
                       </tr>
+                      <tr ng-if="deviceCfg.untrusted">
+                        <th><span class="fa fa-fw fa-user-secret"></span>&nbsp;<span translate>Untrusted</span></th>
```

Comment:
Why not `fas`?

Reply:
`fas` (and `far`) is the outdated syntax. The up-to-date one only uses `fa`. The former only still work because they are included in https://github.com/syncthing/syncthing/blob/main/gui/default/vendor/fork-awesome/css/v5-compat.css 😉 (using the exact same style as `fa`).

https://github.com/syncthing/syncthing/blob/cf46bf029755997899da1d20d5634519c677ec84/gui/default/vendor/fork-awesome/css/fork-awesome.css#L23-L30

vs

https://github.com/syncthing/syncthing/blob/cf46bf029755997899da1d20d5634519c677ec84/gui/default/vendor/fork-awesome/css/v5-compat.css#L13-L22